.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# default codeowners for all files
```

Comment:
Something is going on with these files (they shouldn't be here) -- looks like your rebase went a bit awry. Maybe try it again?

Reply:
Following the same steps as before?

Reply:
Alright, I rebased again, and these files are still here. Suggestions?

Reply:
I'll give it a shot!

Reply:
Your fix looks good to me. Shall we mergr, @karalekas ?